dropshot/src/api_description.rs

```diff
@@ -171,13 +178,31 @@ pub enum ApiEndpointBodyContentType {
     Bytes,
     /** application/json */
     Json,
+    /** application/x-www-form-urlencoded */
+    UrlEncoded,
```

Comment:
In all but the `UntypedBody` case, we do care because we need to know how to decode the body.

Reply:
sorry: I know we care what it is for decoding, but can we stuff that into a string rather than having Json and UrlEncoded as the two enumerated  options?